organization-understanding.md

```diff
@@ -0,0 +1,76 @@
+# Memorandum of Understanding
```

Comment:
We should not use the legal term Memorandum of Understanding. This document is not a contract and we are not looking for a contract.

I would suggest: NumFOCUS GSOC Operations